src/SDKs/MixedReality/MixedReality.Tests/MixedReality.Tests.csproj

```diff
@@ -0,0 +1,34 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
```

Comment:
@XiangyuL-Microsoft model your test project looking at other test projects in the repository.
Take a look at this
https://github.com/Azure/azure-sdk-for-net/blob/master/src/SDKs/Compute/Compute.Tests/Compute.Tests.csproj